vignettes/articles/oauth.Rmd

```diff
@@ -196,13 +235,31 @@ When you run this code, you'll see something like this, but it will obviously co
 #>  $ location     : chr "Houston, TX"
 ```
 
-### Token management
+### Caching
 
 There are two big reasons to allow httr2 to manage tokens for you.
 The first is that httr2 will automatically refresh the token if it's expired.
-The second is cross-session caching which you can turn on with `cache_disk = TRUE`.
-Turning it on means that your users will only need to re-authenticate once a month, but does means their credentials are saved on disk.
-Reading the "Caching" section in `?req_oauth_auth_code` for more details.
+The second is cross-session caching, as described below.
+
+By default, the OAuth token will be cached in memory.
+That means that you will only need to authenticate once in the current session, but you'll need to re-authenticate if you restart R.
+In some cases, you may want to save the tokens (refresh and access) so that they can be used across sessions.
+This is easy to do (just set `cache_disk = TRUE`) but you need to think through the consequences of saving the refresh token on disk.
+
+httr2 does the best it can to save these credentials securely.
+They are stored in a local cache directory (`oauth_cache_path()`) that is accessible to the current user, and they are encrypted so they will be hard for any package other than httr2 to read.
+However, there's no way to prevent other R code from using httr2 to access them, so if you do choose to cache tokens, you should inform the user and give them the ability to opt-out.
+httr2 automatically deletes any cached tokens that are older than 30 days whenever it's loaded.
+This means that you'll need to re-auth at least once a month, but prevents tokens for hanging around on disk long after you've forgotten you created them.
+
+You can see which clients have cached tokens by looking in the cache directory used by httr2:
+
+```{r}
+#| eval: false
+dir(oauth_cache_path(), recursive = TRUE)
+```
+
+Each client gets its own subdirectory named using the client `name`, so if you turn caching on, it's particularly important to give you client a good name so that the user can easily tell with package the tokens belong to.
```

Comment:
```suggestion
Each client gets its own subdirectory named using the client `name`, so if you turn caching on, it's particularly important to give your client a good name so that the user can easily tell which package the tokens belong to.
```